nym-vpn/ui/src-tauri/src/fs/config.rs

```diff
@@ -1,6 +1,17 @@
+use std::path::PathBuf;
+
 use serde::{Deserialize, Serialize};
 
 #[derive(Default, Serialize, Deserialize, Debug, Clone)]
 pub struct AppConfig {
-    // TODO
+    /// Path pointing to an env configuration file describing the network
+    pub env_config_file: Option<PathBuf>,
+    /// Mixnet public ID of the entry gateway
+    pub entry_gateway: String,
+    /// Mixnet recipient address
+    pub exit_router: String,
+    /// NYM API URL
+    /// ⚠ use the same as the one provided by the env_config_file
+    /// TODO use the one provided by the env_config_file
+    pub nym_api: String,
```

Comment:
Let's get rid of this before we merge. This will risk causing lots of confusing errors reports when people start playing around with the app